docs/source/contributing/developing.adoc

```diff
@@ -282,6 +287,12 @@ Set `no-colors` to true to disable ansi colors of Godog's output.
 Definitions::
 Set `definitions` to true to print all available step definitions.
 
+[NOTE]
```

Comment:
@hferentschik Based on our conversation around tags I have added this note, it is connected to addition of target specific variables in this PR.